docs/dev/tools/cli.md

```diff
@@ -279,6 +279,192 @@ viam organizations --help
 
 ## Commands
 
+### `data`
```

Comment:
Oof--quick scan says this is just alphabetizing, trusting no material changes since I don't see any